frontend/app/services/GuardianContentService.scala

```diff
@@ -123,6 +130,20 @@ trait GuardianContent {
     }
   }
 
+  def membershipFrontContentQuery(page: Int): Future[ItemResponse] = {
+    val itemQuery = ItemQuery("/membership")
+      .showFields("all")
+      .showElements("all")
+      .pageSize(50)
```

Comment:
Ditto with showElements - do we just render images as we could just request those?